src/Components/Library/PlaylistSection/PlaylistCard/PlaylistCard.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom';
+
+import {useContext} from 'react'
+import {VideoContext} from '../../../../Store/VideoContext'
+
+
+function PlaylistCard({ current }) {
+    const {videos} = useContext(VideoContext)
+    const currVid = videos.find(item => item.id === current)
```

Comment:
Instead of currVid you can give a meaningful name like currentVideo. Please try to avoid short names as much as you can so that your code can be more readable.

Reply:
Yeah Done!